OSGI-INF/ZWaveBindingConstants.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2014-2016 by the respective copyright holders.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.openhab.binding.zwave.ZWaveBindingConstants">
+   <implementation class="org.openhab.binding.zwave.ZWaveBindingConstants"/>
+
+   <reference bind="setI18nProvider" cardinality="1..1" interface="org.eclipse.smarthome.core.i18n.I18nProvider" name="I18nProvider" policy="static" unbind="unsetI18nProvider"/>
```

Comment:
If you are using 0..1 there is no need to restart this service if the I18nProvider service disappears and appears again (e.g. by a bundle restart etc.). WRT the current bundle location of that service it doesn't really make a difference, but as you can go ahead without the i18n provider service, perhaps you want to change that.